### PR TITLE
Player informations panel

### PR DESCRIPTION
**Pull request by paulSanchezMarmelab**
screen

Part of https://trello.com/c/Lv9Ub5eQ/61-2-as-hayao-i-want-to-observe-an-ongoing-game-between-two-others-players

- [X] Display turn number
- [X] Display message
- [X] Display remaining tiles
- [X] Display current player marker

### CODE REVIEW

**Comment by fzaninotto** [comment]
in English, "n°" is spelled "#", so that would be "Turn #23"

**Comment by fzaninotto** [comment]
Can you rebase on master so that the diff only shows your changes?

**Comment by paulSanchezMarmelab** [comment]
It's merged with main.

File: mobileapp/components/HUD/HUD.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    flex: 1,
+    justifyContent: "center",
+    alignContent: "center",
+  },
+  turnCounterText: {
+    marginBottom: 10,
+    marginTop: -15,
+    fontSize: 20,
+  },
+  messageContainer: {},
```
Comment (adguernier) [inline mobileapp/components/HUD/HUD.tsx:58]: useless style here

File: mobileapp/components/HUD/HUD.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  messageText: {
+    color: "white",
+  },
+});
+
+type hudProps = {
```
Comment (adguernier) [inline mobileapp/components/HUD/HUD.tsx:64]:
```suggestion
type HudProps = {
```

File: mobileapp/components/HUD/HUD.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  gameState: GameState;
+};
+
+export const HUD = (props: hudProps) => {
```
Comment (adguernier) [inline mobileapp/components/HUD/HUD.tsx:68]:
```suggestion
export const HUD = (props: HudProps) => {
```

File: mobileapp/components/HUD/HUD.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    playing: "rgba(209, 209, 0, 0.767)",
+    not_playing: "white",
+  };
+
+  const generatePlayerBlock = (playerName) => {
+    const isCurrentPlayer = gameState.currentPlayer === playerName;
+
+    return (
+      <View style={styles.playerContainer}>
+        <Svg height="100%" width={"100%"}>
```
Comment (guilbill) [inline mobileapp/components/HUD/HUD.tsx:82]: Why one is `"100%"` and the other `{"100%"}`?

**Review by guilbill — APPROVED** [review]

**Review by fzaninotto — APPROVED** [review]

**Review by guilbill — APPROVED** [review]

**Pull request merged**